### PR TITLE
Fix scale factor behavior in `calc()`

### PR DESCRIPTION
**Pull request by RobertasJ**

### CODE REVIEW

File: crates/torin/src/values/size.rs [hunk 1]
```diff
@@ -166,7 +166,10 @@ impl Scaled for Size {
         match self {
             Size::Pixels(s) => *s *= scale_factor,
             Size::DynamicCalculations(calcs) => {
-                calcs.iter_mut().for_each(|calc| calc.scale(scale_factor));
+                calcs.insert(0, DynamicCalculation::OpenParenthesis);
+                calcs.push(DynamicCalculation::ClosedParenthesis);
+                // implicit multiplication
```
Comment (marc2332) [inline crates/torin/src/values/size.rs:171]: Lets make it no implicit for the sake of not under-simplifying

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: crates/torin/src/values/size.rs (hunk 1 above)
Comment (RobertasJ) [reply crates/torin/src/values/size.rs]: Done :D

**Review by marc2332 — CHANGES_REQUESTED** [review]
Can you add some units asserts that check that scaling a size using calc() works just fine with different sizes of scale factors? Also format the code :D

**Review by marc2332 — APPROVED** [review]

**Comment by marc2332** [comment]
Nice work!

**Pull request merged**